romancal/regtest/test_dark_current.py

```diff
@@ -46,28 +46,6 @@ def test_dark_current_outfile_step(rtdata, ignore_asdf_paths):
     assert diff.identical, diff.report()
 
 
-@pytest.mark.bigdata
-def test_dark_current_outfile_suffix(rtdata, ignore_asdf_paths):
-    """Function to run and compare Dark Current subtraction files. Here the
-    test is for renaming the output file."""
-    input_datafile = "r0000101001001001001_01101_0001_WFI01_linearity.asdf"
-    rtdata.get_data(f"WFI/image/{input_datafile}")
-    rtdata.input = input_datafile
-
-    args = [
-        "romancal.step.DarkCurrentStep",
-        rtdata.input,
-        "--output_file=Test_dark",
-        '--suffix="suffix_test"',
-    ]
-    RomanStep.from_cmdline(args)
-    output = "Test_darkcurrent.asdf"
```

Comment:
I chose to delete this test as it appears to check that calling the `DarkCurrentStep` and providing an expected output suffix of `suffix_test` produces a file with a suffix `darkcurrent`.

This isn't the expected behavior for stpipe (the output file should have the suffix provided on the commandline, `suffix_test` not `darkcurrent`).

Reply:
@ddavis-stsci mentioned that this test was intended to test if a user provided suffix is used.

It could be made to pass by:
- updating the expected output from `Test_darkcurrent.asdf` to `Test_suffix_test.asdf` (which includes the user-provided suffix)
- update the regression test file to have the matching `Test_suffix_test.asdf` name

One question might be, why isn't the expected output `Test_dark_suffix_text.asdf` since the user provided `--output_file="Test_dark"` and `--suffix="suffix_test"`? This is due to `dark` being listed as a supported [suffix](https://github.com/spacetelescope/romancal/blob/ebc173004b88b5691ed37007b1344986ac9a0cc0/romancal/lib/suffix.py#L41).